src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_api.py

```diff
@@ -421,8 +424,8 @@ def remove_python_exception_breakpoint(self, py_db, exception):
         except:
             pydev_log.debug("Error while removing exception %s" % sys.exc_info()[0])
 
-        py_db.on_breakpoints_changed(remove=True)
-    
+        py_db.on_breakpoints_changed(removed=True)
```

Comment:
Can you make this a separate PR? We are planning on making a release today and would like to take this.